### PR TITLE
Fix Hebrew translation

### PR DESCRIPTION
**Pull request by alumag**
Fixed some typing and grammar mistakes in Hebrew

### CODE REVIEW

**Comment by phelmig** [comment]
Thank you for your fix! Since we can't validate the contribution's quality we are currently requiring all translations to be reviewed by at least two native speakers (we count you as the contributor as the first). If everything is fine, a simple comment on this pull request is sufficient. If you can reach out to your personal network to find someone for a peer review it would be highly appreciated.

**Review by shahar603 — APPROVED** [review]

**Review by OrenShapira — APPROVED** [review]
Perfect! I'm a Hebrew speaker

File: site/he.njk [hunk 1]
```diff
@@ -103,7 +103,7 @@ lang: he
             <p class="text-2xl">
                 על ידי הצטרפות לתנועה ועל ידי כך הגבלת הדבקות חדשות, אתם לא מגינים רק על עצמכם, אלא גם עוזרים להכיל ולהגביל את ההתפשטות לאחרים, במיוחד לאלו שנמצאים בסיכון גבוה לסיבוכים מהנגיף.<br /><br />
                 
-                <strong>אם איכפת לכם ממטרה זו, אנא שתפו במדיה החברתיות, באימייל או פשוט אימרו לחבריכם ומשפחתכם אודותיה. אבל בכל מקרה, בבקשה רק StayTheFuckHome#.</strong>
+                <strong>אם אכפת לכם ממטרה זו, אנא שתפו במדיות החברתיות, באימייל או פשוט אימרו לחבריכם ומשפחתכם אודותיה. אבל בכל מקרה, בבקשה רק StayTheFuckHome#.</strong>
```
Comment (avrahamshukron) [inline site/he.njk]: מדיה היא צורת רבים, ולכן הביטוי הנכון הוא במדיה החברתית.
```suggestion
                <strong>אם אכפת לכם ממטרה זו, אנא שתפו במדיה החברתית, באימייל או פשוט אימרו לחבריכם ומשפחתכם אודותיה. אבל בכל מקרה, בבקשה רק StayTheFuckHome#.</strong>
```

**Review by avrahamshukron — CHANGES_REQUESTED** [review]
תיקון קל

**Review by MeitarR — APPROVED** [review]
looks good

**Review by avrahamshukron — APPROVED** [review]

**Review by phelmig — APPROVED** [review]
Thank you!

**Pull request merged**